examples/headsail-bsp/mem_hpc.x

```diff
@@ -1,6 +1,6 @@
 MEMORY
 {
-  RAM : ORIGIN = 0x10000, LENGTH = 0x8000
+  RAM : ORIGIN = 0x100010000, LENGTH = 0x8000
```

Comment:
This is wrong as discussed live. HPC has direct access to this one without the external bit. Original version seems correct.

Reply:
mem_hpc.x now only includes the internal bootram of the HPC (0x1_0000)